genestack_client/settings/genestack_config.py

```diff
@@ -49,8 +52,8 @@ def remove_user(self, user):
         self.save()
         try:
             import keyring
-            if keyring.get_password(GENESTACK_SDK, user.alias):
-                keyring.delete_password(GENESTACK_SDK, user.alias)
+            if keyring.get_password(_PASSWORD_KEYRING, user.alias):
+                keyring.delete_password(_PASSWORD_KEYRING, user.alias)
         except ImportError:
```

Comment:
Only `import keyring` should be guarded against `ImportError`, and `Exception` could be thrown in get/delete_password only.